### PR TITLE
style(settings): Added green banner for account recovery

### LINKED ISSUE #7029: Green banner for Account recovery enabled/disabled actions not displayed
Environment: Android, iOS, Win10
Stage - Train 132

Steps to reproduce:
1. Log in and go to Manage account.
2. Go to "Account recovery" and tap on "enable".
3. Generate a new recovery key.
4. Revoke the key.

Expected results: the green banner "Account recovery enabled" is displayed at step 3 and for step 4 - another banner.

Actual results: the green banner is not displayed for step 3 nor step 4. 

Note:
 I've made a video with the issue and also with the "Two-step authentication" section to see how it behaves.

Video:  https://drive.google.com/open?id=1SM3-I3NKcoSZZh4NBTnxM_-hiAQuqTq-

### PR DESCRIPTION
**Pull request by hritvi**
fixes #7029 

**When account recovery enabled:** 
Screenshot from 2019-03-11 14-55-19

**and when it is revoked:**
Screenshot from 2019-03-11 15-01-24

@SorinaFlorean @shane-tomlinson please review :slightly_smiling_face:

### CODE REVIEW

File: app/scripts/views/settings/account_recovery/confirm_password.js [hunk 1]
```diff
@@ -43,6 +45,7 @@ const View = FormView.extend({
     return account.createRecoveryBundle(password)
       .then((result) => {
         this.logFlowEvent('success', this.viewName);
+        this.displaySuccess(t('Account recovery enabled'), {});
```
Comment (shane-tomlinson) [inline app/scripts/views/settings/account_recovery/confirm_password.js]: You shouldn't need the `, {}` here.

File: app/scripts/views/settings/account_recovery/confirm_revoke.js [hunk 2]
```diff
@@ -33,6 +35,7 @@ const View = FormView.extend({
     const account = this.getSignedInAccount();
     return account.deleteRecoveryKey()
       .then(() => {
+        this.displaySuccess(t('Account Recovery Revoked'), {});
```
Comment (shane-tomlinson) [inline app/scripts/views/settings/account_recovery/confirm_revoke.js]: You shouldn't need the `, {}` here.

**Comment by hritvi** [comment]
@shane-tomlinson could you please review it now as all the tests have passed. Thanks :slightly_smiling_face: !!

File: app/scripts/views/settings/account_recovery/confirm_revoke.js [hunk 3]
```diff
@@ -33,6 +35,7 @@ const View = FormView.extend({
     const account = this.getSignedInAccount();
     return account.deleteRecoveryKey()
       .then(() => {
+        this.displaySuccess(t('Account Recovery Revoked'));
```
Comment (shane-tomlinson) [inline app/scripts/views/settings/account_recovery/confirm_revoke.js]: Can you lowercase `Recovery Revoked`?

**Comment by shane-tomlinson** [comment]
Tagging train-134 since this involves string additions and the string cutoff has already occurred.

**Comment by hritvi** [comment]
@shane-tomlinson could you please rerun the circle-ci tests....I am unable to understand why those tests are failing after a mere change of `Account Recovery Revoked` to `Account recovery revoked`.

**Comment by hritvi** [comment]
The tests have passed @shane-tomlinson. Are any more changes needed in this PR?

**Comment by lmorchard** [comment]
I think we should be able to merge this now - [v1.333.0 has been tagged](https://github.com/mozilla/fxa-content-server/releases/tag/v1.133.0)

**Pull request merged**